README.md

```diff
@@ -5,7 +5,7 @@ This repository contains the **source code for [telemetry.mozilla.org](https://t
 
 The main dashboards on [telemetry.mozilla.org](https://telemetry.mozilla.org) consume data from Telemetry's v2 and v4 backends using Telemetry.js. For an overview of how all these systems fit together, see [Telemetry Demystified](https://anthony-zhang.me/blog/telemetry-demystified/).
 
-The dashboards that do not use Telemetry.js generally use [scheduled analysis jobs](https://telemetry-dash.mozilla.org/) that regularly publish data on S3. The source code for these can be found in their respective repositories.
+The dashboards that do not use Telemetry.js generally use [scheduled analysis jobs](https://analysis.telemetry.mozilla.org/) that regularly publish data on S3. The source code for these can be found in their respective repositories.
```

Comment:
Will do. The new hostname hasn't officially been deployed yet, I just figured I'd make the change here while I was changing other stuff.